src/NHSDPublicBrowseAcceptanceTests.Tests/NHSDPublicBrowseAcceptanceTests.Tests.csproj

```diff
@@ -59,4 +67,10 @@
     </SpecFlowFeatureFiles>
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Update="Features\DFOCVC\ViewDFOCVCSolutionsList.feature.cs">
+      <DependentUpon>ViewDFOCVCSolutionsList.feature</DependentUpon>
+    </Compile>
+  </ItemGroup>
```

Comment:
Are all the additions to the project file definitely required? I think I've seen SpecFlow add superfluous entries to project files before.

Reply:
I'll get those removed, never seem to figure out why they are added either